src/routes/matchHistoryRoutes.js

```diff
@@ -0,0 +1,18 @@
+import express from 'express'
+
+function getSummoner(){
+    const router = express.Router()
+    router.get('/summoner', getSummonerHandler)
+    router.get('/summoner/:id', getSummonerNameHandler)
+    return router
+}
+
+async function getSummonerHandler(req, res) {
+    res.json({"Summoner Id": "This is the summoner endpoint!"})
+}
+
+async function getSummonerNameHandler(req, res){
+    res.json({"message": req.params.id})
```

Comment:
Instead of returning the just the path parameter value, use that value in a message. I'll let you figure out how to do this, but return a message that says "Welcome, <summonerName>!!".

Note that the request parameter name has changed from id to summoner.